src/magicbox/SuggestionsManager.ts

```diff
@@ -128,7 +130,7 @@ export class SuggestionsManager {
 
   public selectAndReturnKeyboardFocusedElement(): HTMLElement {
     const selected = this.keyboardFocusedElement;
-    if (selected != null) {
+    if (!!selected) {
```

Comment:
I'm seeing the double `!!` here and on line 250 in the previous file. Any reason not to just do:
```
if (selected) {
    ...
}
```
and

```
return find(this.lastSuggestions, suggestion => suggestion.text);
```

Reply:
Yes, `!!` is very useful for casting to a boolean, but a `if` will already evaluate the condition to see it's "truthy" (here selected). So it will check if it's not `null`, `undefined`, `0`, etc.

Reply:
Woops, I just went by fast converting them all without paying much attention.

It's the kind of thing that should've already been a habit for me by now, given that before Javascript I programmed in Lua, which evaluated the exact same types as `true`...

### Tl;dr: Fixing it.